packages/expo/src/generators/preset/preset.ts

```diff
@@ -0,0 +1,10 @@
+import { Tree } from '@nrwl/devkit';
+import expoApplicationGenerator from '../application/application';
+import { PresetGeneratorSchema } from './schema';
+
+export default async function expoPresetGenerator(
+  tree: Tree,
+  options: PresetGeneratorSchema
+) {
+  await expoApplicationGenerator(tree, options);
```

Comment:
The name in options is the workspace name I believe. Should use `--appName` or another option.

```
npx -y create-nx-workspace my-workspace --preset=@nrwl/expo --appName=my-app
```

The `--appName` option is what we usually pass to the command. There was a bug with custom presets too  when trying to pass the app name, should double-check that it works.